applications/nrf5340_audio/prj_release.conf

```diff
@@ -13,3 +16,9 @@ CONFIG_SERIAL=n
 CONFIG_CONSOLE=n
 CONFIG_PRINTK=n
 CONFIG_UART_CONSOLE=n
+
+# USB
+# Default values from src/modules/Kconfig.defaults.
+# Must be changed before production.
+# CONFIG_USB_DEVICE_VID=0x1915
```

Comment:
Would this cause warning during the build?
Since I was going to add "#CONFIG_BT_USE_DEBUG_KEY=y" in the prj.conf in the before but heard that the build system will check if there's any commented config in the .conf.
Let me make a trial later :P

Reply:
I talked to Torsten, and he actually said that this is probably the way to solve it (to expose important configs that have to be changed before production).
Maybe CONFIG_BT_PRIVACY=y should be added here as well?